erpnext/stock/doctype/price_list/test_price_list.py

```
@@ -2,3 +2,23 @@
 # License: GNU General Public License v3. See license.txt
 
 
+import frappe
+from frappe.tests import IntegrationTestCase
+
+
+class TestPriceList(IntegrationTestCase):
+	pass
+
+
+def create_price_list_for_default_check(self, price_list_name, currency="INR", is_default=0):
```

Ruff (0.14.13)

Unused function argument: `self`

(ARG001)